mne/time_frequency/tests/test_spectrum.py

```diff
@@ -270,12 +271,14 @@ def test_spectrum_kwarg_triaging(raw):
     import matplotlib.pyplot as plt
 
     regex = r"legacy plot_psd\(\) method.*unexpected keyword.*'axes'.*Try rewriting"
-    fig, axes = plt.subplots(1, 2)
+    _, axes = plt.subplots(1, 2)
     # `axes` is the new param name: technically only valid for Spectrum.plot()
     with pytest.warns(RuntimeWarning, match=regex):
         raw.plot_psd(axes=axes)
     # `ax` is the correct legacy param name
-    raw.plot_psd(ax=axes)
+    with warnings.catch_warnings():
+        warnings.simplefilter(action="ignore", category=FutureWarning)
+        raw.plot_psd(ax=axes)
```

Comment:
here again, is there a reason why not `pytest.warns`? It has the advantage of failing when the warning goes away, which prompts us to update the test at the appropriate time.